tests/utest

```diff
@@ -182,116 +270,145 @@ run_utests() {
 					export LIST_FILES="${LIST_FILES}"
 					EOF
         ;;
-      32bit)
-        comp_nm=$rc
+      bit_base)
```

Comment:
Is the bit_base run to create 32bit baseline and then the bit test runs again the baseline?

Reply:
By default, `Bit' and `Debug' only check if code compiles and runs to completion. There is, however, an option to check the reproducibility of Bit (-b) and Debug (-d).